package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "OAS (Swagger v3) API Definition for Template API",
   "scripts": {
-    "lint": "node_modules/.bin/speccy lint specification/e-referrals-service-api.yaml --skip default-and-example-are-redundant",
+    "lint": "node_modules/.bin/speccy lint -s openapi-tags -s operation-tags specification/e-referrals-service-api.yaml --skip default-and-example-are-redundant",
```

Comment:
I think the "-s" flag details that Speccy should skip a given rule, how come we're having to skip the "openapi-tags" and "operation-tags" rules?

Reply:
as we don't use tags currently. Similar thing has been done on PDS project. IF we decide to use tags for grouping this rule omission can be taken out.